### ISSUE TITLE
Add --build-arg support

### ISSUE THREAD

**Issue by ShadowCoder22** [Transmode/gradle-docker#91]
Docker allows users to pass arguments to the build command. This is useful for cases where a developer has to calculate a variable at build time. Please consider adding support for this feature.

**Comment by bjornmagnusson** [comment]
As I see it, only reason for using this property would be when using an static/external Dockerfile. When plugin is generating the Dockerfile you already can provide properties from within you build script

**Comment by bjornmagnusson** [comment]
Unsupported in docker-java 2.2.0 (used by plugin in master). Needs docker-java 3.0.0+, which includes breaking changes. Postpone this to an later release.

**Comment by gclayburg** [comment]
It would be nice to have this so that you could add build time arguments to a Dockerfile LABEL like this

```
docker build --build-arg VCS_REF=`git rev-parse --short HEAD` .
```

**Comment by chadlwilson** [comment]
Yes; or to pass parameters to other Dockerfile elements that the plugin doesn't have support for; such as `HEALTHCHECK`

**Comment by Mklueh** [comment]
News on this? Need to disable the cache

**Comment by bjornmagnusson** [comment]
No work has been done. It is indeed a wanted feature 
We do welcome PRs.

**Comment by jainarpit6** [comment]
Any progress on this ?
